### ISSUE TITLE
fig incompatible

### ISSUE THREAD

**Issue by ramziosta** [withfig/fig#2742]
### Checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have run `fig restart` and replicated the issue again

### Operating system

macOS 13.5.2 (22G91)

### Expected behaviour

fig incompatible error with pycharm 17.0.8+7-b1000.8 aarch64

### Actual behaviour

no aparent issue, cli works as expected, just an error when I log in to p

### Steps to reproduce

every time I run pycharm I get the incompatible error. The terminal works fine with fig running properly

### Environment

```yaml
fig-details:
  - 2.16.0
hardware-info:
  - model: 
  - model-id: 
  - chip-id: Apple M1
  - cores: 8
  - mem: 16.00 GB
os-info:
  - macOS 13.5.2 (22G91)
environment:
  - shell: /bin/zsh
  - terminal: intellij-pycharm-ce
  - cwd: /Users/ramziosta
  - exe-path: /opt/homebrew/bin/fig
  - install-method: brew
  - env-vars:
    - FIGTERM_SESSION_ID: 85765514-d058-4354-83dc-6f7d0173b297
    - FIG_JETBRAINS_SHELL_INTEGRATION: 1
    - FIG_SET_PARENT_CHECK: 1
    - FIG_TERM: 2.16.0
    - PATH: /Library/Frameworks/Python.framework/Versions/3.11/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/Frameworks/Mono.framework/Versions/Current/Commands:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin:/Users/ramziosta/.fig/bin:/Users/ramziosta/.local/bin
    - SHELL: /bin/zsh
    - TERM: xterm-256color
    - __CFBundleIdentifier: com.jetbrains.pycharm.ce
    - FIG_PID: 1778
    - FIG_SET_PARENT: 85765514-d058-4354-83dc-6f7d0173b297
```

**Comment by mschrage** [comment]
Looks like the same issue #2052. We are investigating.

**Comment by mschrage** [comment]
Closing as a duplicate

**Comment by ramziosta** [comment]
It is still happening, I have a current pycharm and webstorm


On Sep 26, 2023, at 7:08 PM, Matt Schrage ***@***.***> wrote:



Closed #2742<https://github.com/withfig/fig/issues/2742> as not planned.
